src/__tests__/lib/containers/UserCardTests.tsx

```diff
@@ -156,11 +155,9 @@ describe('it creates the correct UI for the medium card', () => {
       }
     ] as MenuAction []
     const { wrapper } = createMediumComponent({ ...props, menuActions })
-    expect(wrapper.render().find('SRC-menu-wall')).toHaveLength(0)
```

Comment:
looks like "." was missing from SRC-menu-wall class selector, adding that back in to be ".SRC-menu-wall" fixed the issue, I think we could keep check in there.

Reply:
Good catch, I'll re-add it!